docs/release-notes/NuGet-5.5.md

```diff
@@ -0,0 +1,168 @@
+
+* Logic in RestoreTask should be refactored into a utility - [#8829](https://github.com/NuGet/Home/issues/8829)
+
+* Deadlock in VSSettings init - [#8842](https://github.com/NuGet/Home/issues/8842)
+
+* "An item with the same key has already been added" when restoring in Visual Studio - [#8847](https://github.com/NuGet/Home/issues/8847)
+
+* DownloadTimeoutStream should not block by calling .Result - [#8853](https://github.com/NuGet/Home/issues/8853)
+
+* RestoreSettingsUtils needs overload to use new SettingsLoadingContext - [#8855](https://github.com/NuGet/Home/issues/8855)
+
+* VisualStudio ToolBox is not populated from a NuGet package if a project is placed in a solution folder - [#8868](https://github.com/NuGet/Home/issues/8868)
+
+* VS:  solution restore perpetually fails due to race condition - [#8881](https://github.com/NuGet/Home/issues/8881)
+
+* Constant "loading.." on installed tab, and "searching <term>.." on updates tab - [#8890](https://github.com/NuGet/Home/issues/8890)
+
+* ProjectReference becomes dependency in nuspec instead of including it in nupkg - [#8960](https://github.com/NuGet/Home/issues/8960)
+
+* Update the NuGet VS depedencies to the recent 16.4 ones so we can use the Service Broker APIs. - [#8967](https://github.com/NuGet/Home/issues/8967)
+
+* Address VSSDK004, VSSDK005 & VSSDK006 analyzer warnings - [#8974](https://github.com/NuGet/Home/issues/8974)
+
+* Address VSTHRD100 - [#8975](https://github.com/NuGet/Home/issues/8975)
+
+* Address VSTHRD010 and VSTHRD109 warnings - [#8978](https://github.com/NuGet/Home/issues/8978)
+
+* NuGet client tools strategy with multiple feeds on packages with version floats - [#8992](https://github.com/NuGet/Home/issues/8992)
+
+* hashed package names in telemetry should be lower cased - [#8995](https://github.com/NuGet/Home/issues/8995)
+
+* Accessibility Fixes in PM UI - altText, etc... - [#9059](https://github.com/NuGet/Home/issues/9059)
+
+* Missing Embedded Icons in VS PM UI after cache expires - [#9069](https://github.com/NuGet/Home/issues/9069)
+
+* Icon missing after package install in VS PM UI - [#9072](https://github.com/NuGet/Home/issues/9072)
+
+* Review Accessibility bugs - [#9077](https://github.com/NuGet/Home/issues/9077)
+
+* PMUI Tabs have Accessibility Issues - [#9078](https://github.com/NuGet/Home/issues/9078)
+
+* Review JoinableTaskFactory.Run( usage - [#9096](https://github.com/NuGet/Home/issues/9096)
+
+* FireAndForget PM UI startup - [#9112](https://github.com/NuGet/Home/issues/9112)
+
+* Missing Icon in PM UI Details Pane - [#9113](https://github.com/NuGet/Home/issues/9113)
+
+* Accessibility Fixes in PM UI - [#9157](https://github.com/NuGet/Home/issues/9157)
+
+* Restore:  IncludeExcludeFiles.Equals(...) implementation is incorrect - [#9167](https://github.com/NuGet/Home/issues/9167)
+
+* Restore:  PackageSpec.Clone() creates unequal clone - [#9211](https://github.com/NuGet/Home/issues/9211)
+
+**Feature:**
```

Comment:
Feature should be above bug, ordered by importance.

Reply:
Next commit - modified the release notes in the following order

- Feature
- DCR
- Bugs
- None (still need to look into the appropriate `Type`label for these issues)